cypress/integration/T3.js

```diff
@@ -0,0 +1,38 @@
+import TaskPage from '../objects/taskPage';
+import Common from '../objects/common';
+
+const taskPage = new TaskPage();
+const common = new Common();
+
+describe('Task creation', function() {
+
+    beforeEach(function () {
+        cy.loginAs('Admin');
+        common.visit('/Tasks');
+    })
+
+    it('Admin creates a new task', function () {
+        
+        common.clickCertainButton('button','Create Task');
+        const taskName = Date.now().toString(); 
+        taskPage.inputIntoASpecificField('taskDetailsForm.name', taskName);
```

Comment:
taskDetailsForm.name is an ID of the field and it is stored in a test case, so this approach does not follow page object model. If this field id changes to something else, you would still need to go through each of your test case and change it manually